apps/builder/app/env/env.public.server.ts

```diff
@@ -28,9 +28,10 @@ const env = {
   PUBLISHER_HOST: process.env.PUBLISHER_HOST || null,
   BUILD_REQUIRE_SUBDOMAIN: process.env.BUILD_REQUIRE_SUBDOMAIN === "true",
 
-  // Must be set for Vercel deployments
-  RESIZE_ORIGIN: process.env.RESIZE_ORIGIN,
-
+  IMAGE_BASE_URL:
```

Comment:
Having proxy this doesn't look right. The idea that from the client we always have image/assets `src`
like 
```jsx
<img src={`/cdn-cgi/image/${resizeoptions}/${name}` />
```

Reply:
So in env.public this should be a `_PATH`